Content.Server/Objectives/Commands.cs

```diff
@@ -0,0 +1,123 @@
+﻿#nullable enable
+using System.Linq;
+using Content.Server.Players;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Objectives
+{
+    public class AddObjectiveCommand : IClientCommand
+    {
+        public string Command => "addobjective";
+        public string Description => "Adds an objective to the player's mind.";
+        public string Help => "addobjective <username> <objectiveID>";
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            if (args.Length != 2)
+            {
+                shell.SendText(player, "Expected exactly 2 arguments.");
+                return;
+            }
+
+            var mgr = IoCManager.Resolve<IPlayerManager>();
+            if (mgr.TryGetPlayerDataByUsername(args[0], out var data))
```

Comment:
I'd rather like a !mgr.TryGet so we don't wrap this in this long if but it's not important